server/config.ts

```diff
@@ -106,6 +106,9 @@ export default {
   variations: {
     enabled: get('VARIATIONS_ENABLED', 'false') === 'true',
   },
+  monitroingContionTimes: {
```

Comment:
this should probably be refactored to "monitoringConditionTimes"

Reply:
Sorry, closed before I see the comment, I will fix this in next PR